src/main/java/com/griefcraft/model/Protection.java

```diff
@@ -1045,14 +1057,14 @@ public void sendProtectionInfo(CommandSender sender) {
                 .timeToString((System.currentTimeMillis() / 1000L)
                         - this.lastAccessed);
 
-        /*
-        TODO: Translate
-        if (!lastAccessed.equals("Not yet known")) {}
-        */
+        String lastAccessedLocale = lwc.getLocaleMessage(sender, "lwc.time.past", "time", lastAccessed)[0];
+        if (lastAccessed.equals("Not yet known")) {
+            lastAccessedLocale = lwc.getLocaleMessage(sender, "lwc.time.unknown")[0];
+        } else if (lastAccessed.equals("less than a second")) {
+            lastAccessedLocale = lwc.getLocaleMessage("lwc.time.early")[0];
+        }
 
-        BlockCache blockCache = BlockCache.getInstance();
-        Material blockType = blockCache.getBlockType(blockId);
-        lwc.sendLocale(sender, "protection.info", "id", id, "type", getType(), "name", UUIDRegistry.getName(UUID.fromString(getOwner())), "owner", getOwner(), "world", world, "x", x, "y", y, "z", z, "creation", creation, "flag", flagStr.toString(), "lastAccessed", lastAccessed);
+        lwc.sendLocale(sender, "protection.interact.info.formatted", "id", id, "type", lwc.getPlugin().getMessageParser().parseMessage(getType().toString().toLowerCase()), "name", UUIDRegistry.getName(UUID.fromString(getOwner())), "owner", getOwner(), "world", world, "x", x, "y", y, "z", z, "creation", creation, "flag", flagStr.toString(), "lastAccessed", lastAccessedLocale);
```

Comment:
Is there a reason why you are avoiding using the Protection::typeToString() and LWC::materialToString(Material) methods? It's used for the raw protection info in the toString method but I don't see why you would need to do anything differently here. This also currently does not display the block that is protected (so it shows less information than the raw protection info).